src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0
```

Comment:
A GA CLI release would be version 2.0.0. and requires a complete command review.

Reply:
My recommendation would be to simply keep the module in preview.

Reply:
That is fine. I reverted to 0.2.0.